### DISCUSSION TITLE
closed

### DISCUSSION THREAD

**Discussion by Musicminion** [Melledy/Grasscutter#2050] [General]
closed

**Comment by weisd** [comment]
https://gc.my.com  是不是用ip可以吗？ http 可以吗？

**Comment by gucooing** [comment]
外置代理就是如mitm、w2、fd这种，是可以使用的，但是需要修改小火箭模块和添加服务器节点，有外置的代理不使用小火箭也是可行的，不过受限于wifi环境，成功率也不高

**Reply by weisd** [reply]
我用小火箭成功下载资源了，就是把sdk.mihoyu.cn修改成自己的服务器ip的时候就不行了，报4201， 是不能用https://ip吗？

**Reply by Someijam** [reply]
> 我用小火箭成功下载资源了，就是把sdk.mihoyu.cn修改成自己的服务器ip的时候就不行了，报4201， 是不能用https://ip吗？

我同样在下载完成之后遇到了这个问题，使用mitmproxy代理也会报4201

**Reply by weisd** [reply]
> > 我用小火箭成功下载资源了，就是把sdk.mihoyu.cn修改成自己的服务器ip的时候就不行了，报4201， 是不能用[https://ip吗？](https://ip%E5%90%97%EF%BC%9F)
> 
> 我同样在下载完成之后遇到了这个问题，使用mitmproxy代理也会报4201
我用下面这个模块成功了
```
#!name=原神 Redirect
#!desc=Redirect

[URL Rewrite]
"[\w-]*\.hoyoverse\.com" xxx.xxx.xxx.xxx  307
"[\w-]*\.mihoyo\.com" xxx.xxx.xxx.xxx 307
"[\w-]*\.yuanshen\.com" xxx.xxx.xxx.xxx 307

[MITM]
hostname=%APPEND% *.mihoyo.com,*.hoyoverse.com,*.yuanshen.com
```

**Reply by Someijam** [reply]
感觉是那个GC iOS客户端的问题，我用官方的国际服客户端可以登录在服务端用命令创建的账号，登陆完成之后会报4214（正常，因为官方未修改的不行），但是切到这里提到的GC iOS客户端就没法跳出登陆界面，报4201。

**Reply by gray19950103** [reply]
> > > 我用小火箭成功下载资源了，就是把sdk.mihoyu.cn修改成自己的服务器ip的时候就不行了，报4201， 是不能用[https://ip吗？](https://ip%E5%90%97%EF%BC%9F)
> > 
> > 
> > 我同样在下载完成之后遇到了这个问题，使用mitmproxy代理也会报4201
> > 我用下面这个模块成功了
> 
> ```
> #!name=原神 Redirect
> #!desc=Redirect
> 
> [URL Rewrite]
> "[\w-]*\.hoyoverse\.com" xxx.xxx.xxx.xxx  307
> "[\w-]*\.mihoyo\.com" xxx.xxx.xxx.xxx 307
> "[\w-]*\.yuanshen\.com" xxx.xxx.xxx.xxx 307
> 
> [MITM]
> hostname=%APPEND% *.mihoyo.com,*.hoyoverse.com,*.yuanshen.com
> ```

请问现在下载资源的服务器正常吗？我这边能够过登录界面。可是点击登陆之后显示网络超时，无法下载资源。3.5;3.4的端都是这个样子

**Comment by edwad919** [comment]
你好 可不可以教一下怎么自己搭建gc服务器 小白有些不懂 搜不到教程

**Comment by HaltCatch** [comment]
网盘链接挂了

**Reply by HaltCatch** [reply]
这个慢的很，算了慢慢下吧。然后想问一下，GC是不是必须要用修改过的IPA，只是砸壳的不能用吗？
